src/mail/mjml/reset_password.mjml

```diff
@@ -0,0 +1,43 @@
+<mjml>
+  <mj-include path="./partial/header.mjml" />
+
+  <mj-body mj-class="bg--blue-100">
+    <mj-wrapper css-class="wrapper" padding="0 40px 40px 40px">
+      <mj-section>
+        <mj-column>
+            <mj-image align="center" src="{% base64_static 'images/messagerie.png' %}" width="60px" height="60px" alt="{% trans 'La Messagerie' %}" />
+        </mj-column>
+      </mj-section>
+      <mj-section mj-class="bg--white-100" padding="0px 20px">
+        <mj-column>
+          <!-- Message -->
+          <mj-text font-size="30px"><b>{% trans "Your password has been reset" %}</b></mj-text>
+          <!-- Main Message -->
+          <mj-text>{% trans "Your password has been reset." %}</mj-text>
+          <mj-text>{% trans "Please find below your new login information: " %}</mj-text>
+        </mj-column>
+      </mj-section>
+      <mj-section background-color="#f3f2fe" padding="0px 20px">
+        <mj-column>
+          <mj-text>{% trans "Email address: "%}<b>{{ mailbox_data.email }}</b></mj-text>
+          <mj-text>{% trans "Temporary password (to be modify on first login): "%}<b>{{ mailbox_data.password }}</b></mj-text>
```

Comment:
Note for later: we should not provide a temporary password, we should use magic link to enforce user to set their password (most users don't update temporary password if not forced to).